components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/APIAdmin.java

```diff
@@ -531,6 +531,52 @@ boolean isScopeExists(String username, String scopeName)
 
     String getTenantConfig(String organization) throws APIManagementException;
 
+    /**
+     * Imports a drafted organization theme for the given organization.
+     *
+     * @param organization Organization name.
+     * @param themeContent Theme content as InputStream.
+     * @throws APIManagementException If a database error occurs.
+     */
+    void importDraftedOrgTheme(String organization, InputStream themeContent) throws APIManagementException;
+
+    /**
+     * Updates the organization theme status as published or unpublished.
+     *
+     * @param organization Organization name.
+     * @param action       Action to perform ("PUBLISH" or "UNPUBLISH").
+     * @throws APIManagementException If a database error occurs.
+     */
+    void updateOrgThemeStatusAsPublishedOrUnpublished(String organization, String action) throws APIManagementException;
```

Comment:
Shall we rename as updateOrgThemeStatus?

Reply:
Ack. Addressed by [5th set of required changes](https://github.com/wso2/carbon-apimgt/pull/12874/commits/12b7446b6a2eaef31588306ac085cba47128277a)